requirements.txt

```diff
@@ -1,18 +1,18 @@
-h5py==2.10.0
+h5py>=2.10.0,<3
 jupyter>=1.0.0,<2
-jupyter_contrib_nbextensions==0.5.1
-matplotlib==2.2.2
+jupyter_contrib_nbextensions>=0.5.1,<1
+matplotlib>=2.2.2,<3
 git+git://github.com/ionpath/mibilib@v1.2.6
```

Comment:
is there a way to add a range here?

Reply:
For now, I do not see a way. It would seem like a perk that only comes if you've integrated your project with PyPI. If anyone knows something about this, would really appreciate it!